piperider_cli/workspace.py

```diff
@@ -576,10 +579,24 @@ def _validate_input_result(result):
     return True
 
 
+def setup_report_varaibles(template_html: str, is_single: bool, data):
+    if isinstance(data, dict):
+        output = json.dumps(data)
+    else:
+        output = data
+    if is_single:
+        variables = f'<script>\nwindow.PIPERIDER_SINGLE_REPORT_DATA={output};window.PIPERIDER_COMPARISON_REPORT_DATA="";</script>'
+    else:
+        variables = f'<script>\nwindow.PIPERIDER_SINGLE_REPORT_DATA="";window.PIPERIDER_COMPARISON_REPORT_DATA={output};</script>'
+    html_parts = re.sub(r'<script>.+</script>', '#PLACEHOLDER#', template_html).split('#PLACEHOLDER#')
```

Comment:
It might be risky if there are two or more `script` tags in the html template. Should we worry about that?

Reply:
Would be best if the react-scripts build from FE didn't add the commas to begin with.
Will try to find solution there (then we won't need the scripts)

Else, may need `id` on the script tag to designate the correct selection

Reply:
Couldn't find a quick way to handle it without ejecting and configuring react-scripts. 

As for the regex approach, could we use bs4 instead to parse the contents? 
https://stackoverflow.com/questions/53608339/how-to-get-a-html-tag-value-using-re
wdyt? @ctiml @qrtt1 

Maybe
```python
import BeautifulSoup
soup = BeautifulSoup.BeautifulSoup('<html><body><div><div id="articlebody"> ... </div></div></body></html')
soup.find("div", {"id": "articlebody"})
```

Reply:
Probably just easier this way. 
![image](https://user-images.githubusercontent.com/11186480/174694742-2a80c9ae-0209-453b-80dd-93302fefe510.png)

Reply:
It is a good idea, but we should consider this case breaking the html

```html
<body>
<div id="root"></div>
<script>window.PIPERIDER_SINGLE_REPORT_DATA = "", window.PIPERIDER_COMPARISON_REPORT_DATA = ""</script>
<script>window.FOOBARBAR = "";</script>
</body>
```

The second `script` pair will be gone, because the default greedy matching by regex. Please make it `lazy`